sashimi-webapp/src/helpers/elementUtils.js

```diff
@@ -10,6 +10,9 @@ export default {
     if (typeof targetReference === 'string') {
       const doc = altDocument || document;
       return doc.getElementById(targetReference);
+    } else if (targetReference.window === targetReference) {
```

Comment:
How come an Element Object has a window properties? Is the jsdoc updated? Thank you!